src/widgets/component-new-edit/components/response-format/table/input-measure.jsx

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import { Field } from 'redux-form';
 import PropTypes from 'prop-types';
 
-import { InputWithVariableAutoCompletion } from 'forms/controls/control-with-suggestions';
+import { RichTextareaWithVariableAutoCompletion } from 'forms/controls/control-with-suggestions';
+import {toolbarConfigTooltip} from "forms/controls/rich-textarea";
```

Comment:
You should use single quote.

Reply:
Done!
I didn't notice, thank you for noticing!